docs: updates for VM Migration Assistant v1.0.5 (#5113)
spectrocloud/librarium: docs/docs-content/vm-management/vm-migration-assistant/additional-configuration.md

| ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| **URL**                  | Your vSphere / ESXi API endpoint for the SDK. You can specify a Full Qualified Domain Name (FQDN) or an IP address. For example, `https://vcenter.mycompany.com/sdk`.                                                                                                                       |
| **External web UI link** | Your vSphere / ESXi UI endpoint. You can specify a Full Qualified Domain Name (FQDN) or an IP address. For example, `https://vcenter.mycompany.com/ui`.                                                                                                                                     |

> | **Convert Disk**         | When enabled, disk conversion is handled using virt-v2v. For example, if you're migrating from VMware to Kernel Virtual Machine (KVM), virt-v2v can convert Virtual Machine Disk (VMDK) to raw or QEMU copy-on-write version 2 (qcow2) formats that are optimal for the target environment. |

Reviewer: ```suggestion
| **Convert Disk**         | When enabled, disk conversion is handled using `virt-v2v`. For example, if you're migrating from VMware to Kernel Virtual Machine (KVM), `virt-v2v` can convert Virtual Machine Disk (VMDK) to raw or QEMU copy-on-write version 2 `qcow2` formats that are optimal for the target environment. |
```

Reviewer: What do you think about putting these in backticks?

Author: Yes for virt-v2v, but not for qcow2.

Author: ```suggestion
| **Convert Disk**         | When enabled, disk conversion is handled using `virt-v2v`. For example, if you're migrating from VMware to Kernel Virtual Machine (KVM), `virt-v2v` can convert Virtual Machine Disk (VMDK) to raw or QEMU copy-on-write version 2 (qcow2) formats that are optimal for the target environment. |
```

> | **Convert Disk**         | When enabled, disk conversion is handled using `virt-v2v`. For example, if you're migrating from VMware to Kernel Virtual Machine (KVM), `virt-v2v` can convert Virtual Machine Disk (VMDK) to raw or QEMU copy-on-write version 2 (qcow2) formats that are optimal for the target environment. |

Reviewer 2: ```suggestion
| **Convert Disk**         | When enabled, disk conversion is handled using `virt-v2v`. For example, if you're migrating from VMware to VMO, `virt-v2v` can convert Virtual Machine Disk (VMDK) to raw or QEMU copy-on-write version 2 (qcow2) formats that are optimal for the target environment. |
```
